rusk-abi/src/dusk.rs

```diff
@@ -0,0 +1,48 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+//! Dusk denomination.
+
+const DUSK_UNIT: f64 = 1_000_000_000.0;
+
+/// The minimum increment of Dusk.
+pub const LUX: Dusk = dusk(1.0 / DUSK_UNIT);
+
+/// The Dusk denomination. Use the [`dusk`] function to convert from floating
+/// point format, and the [`FromDusk`] trait to convert back.
```

Comment:
I believe the comment is a leftover, since we don't have the `FromDusk` trait anymore.